### PR TITLE
party versioning

### PR DESCRIPTION
**Pull request by tej99**
-data model changes for party versioning 
-add alembic to manage party versioning
-returning sampleSummaryId in json response for business/party details

### CODE REVIEW

File: alembic/versions/92464d9a418f_set_up_database.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    sql_query = "INSERT INTO partysvc.business_attributes (business_id, attributes) " \
+                "SELECT party_uuid, attributes FROM partysvc.business"
+    conn.execute(sql_query)
+
+    op.drop_column('business', 'attributes', schema='partysvc')
+
+
+def downgrade():
+    pass
```
Comment (rjweeks) [inline alembic/versions/92464d9a418f_set_up_database.py]: Should we include the downgrade?

**Review by rjweeks — APPROVED** [review]

File: Pipfile [hunk 2]
```diff
@@ -28,7 +28,7 @@ structlog = "==17.2.0"
 flask-httpauth = "==3.2.3"
 ras-common-utils = {git = "https://github.com/ONSdigital/ras-common-utils.git", ref = "0.0.7"}
 retrying = "*"
-
+alembic = "*"
```
Comment (emilioward) [inline Pipfile]: Does this need to be a more specific version of alembic?

**Review by emilioward — APPROVED** [review]
Looks good tests pass

**Review by JamesGardiner — APPROVED** [review]

**Review by jcox-dev — CHANGES_REQUESTED** [review]
Do we need a few simple steps added to the README mentioning that the repo uses alembic?

File: migrations/README [hunk 3]
```diff
@@ -1 +1,14 @@
...
+Upgrade to the most recent version
+```
+alembic upgrade head
+```
+
+
+## Downgrade
+Downgrade to a particular version
+```
+alembic upgrade {version}
```
Comment (jcox-dev) [inline migrations/README]: "Or to undo all migrations: `alembic downgrade base`" might be worth adding in.

**Review by jcox-dev — CHANGES_REQUESTED** [review]

**Review by jcox-dev — APPROVED** [review]

File: migrations/env.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# override sqlalchemy.url
+config_path = 'config/config.yaml'
+with open(config_path, 'r') as f:
+    config_data = yaml.load(f)
```
Comment (andrewmil) [inline migrations/env.py:21]: this should be `yaml.safe_load(f)`

Using `yaml.load` potentially allows arbirtary code execution 
https://security.openstack.org/guidelines/dg_avoid-dangerous-input-parsing-libraries.html

**Review by andrewmil — CHANGES_REQUESTED** [review]
We shouldn't use `yaml.load()`

File: migrations/env.py (hunk 4 above)
Comment (tej99) [reply migrations/env.py:21]: We won't be using yaml. Once @jcox-dev's config refactor goes in, so i'll probably just leave this in for now.

**Review by andrewmil — APPROVED** [review]

**Pull request merged**